app/src/pages/inside/common/filterEntitiesGroups/launchLevelEntities.jsx

```diff
@@ -76,13 +75,9 @@ const messages = defineMessages({
     id: 'LaunchLevelEntities.OwnerTitle',
     defaultMessage: 'Owner',
   },
-  AttributeKeysTitle: {
-    id: 'LaunchLevelEntities.AttributeKeysTitle',
-    defaultMessage: 'Attribute keys',
-  },
```

Comment:
Please, make sure you delete this translations from corresponding langs. Just run `npm run manage:translations`